PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/connector/database/DatabaseResourcePanel.java

```diff
@@ -226,9 +230,9 @@ public DatabaseResource getData() {
         password.password(inputPasswordField.getPassword());
         password.saveType(passwordSaveComboBox.getValue());
 
-        final DatabaseResource resource = new DatabaseResource(definition.getType(),
-                definition instanceof SqlServerDatabaseResource.Definition ?
-                ((SqlDatabaseEntity) databaseComboBox.getValue()).getId() : ((MySqlDatabaseEntity) databaseComboBox.getValue()).getId());
+        final DatabaseResource resource = definition instanceof SqlServerDatabaseResource.Definition ?
+                new SqlServerDatabaseResource(((SqlDatabaseEntity) databaseComboBox.getValue()).getId()) :
```

Comment:
can we force databaseComboBox getValue returns IAzureResourceEntity?

Reply:
no. will extract a IDatabaseResourceEntity to refactor this codes in the Database rewfactor task/PR very soon.